### PR TITLE
Miscellaneous cleanup

### PR DESCRIPTION
**Pull request by wjt**
Prep work for https://phabricator.endlessm.com/T35247.

### CODE REVIEW

File: debian/control [hunk 1]
```diff
@@ -29,6 +29,7 @@ Build-Depends:
  meson (>= 0.60.0),
  ostree (>= 2017.12),
  python3-gi,
+ reuse,
```
[the remark this answers is not in the document]
Comment (wjt) [reply debian/control:32]: I understand the problem. I filed https://phabricator.endlessm.com/T35253 and will just drop this patch for now.

**Comment by wjt** [comment]
Retest this please

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,14 @@
+/debian/*.debhelper.log
```
Comment (dbnicholson) [inline .gitignore:1]: Hilariously, you have been shamed by the reuse gods you are trying to appease:

```
>>> MALLOC_PERTURB_=186 /usr/bin/reuse lint
――――――――――――――――――――――――――――――――――――― ✀  ―――――――――――――――――――――――――――――――――――――
# MISSING COPYRIGHT AND LICENSING INFORMATION

The following files have no copyright and licensing information:
* .gitignore
```

That seems pretty pedantic.

**Review by dbnicholson — APPROVED** [review]
This all looks good assuming you can license the `.gitignore` file or figure out how to make `reuse` not make you have one.

File: .gitignore (hunk 2 above)
Comment (wjt) [reply .gitignore:1]: It's beautiful.

I regret trying to make this work.

Comment (wjt) [reply .gitignore:3]: The options are:

1. Add a license comment to this obviously non-creative work.
2. Add the same information to `.reuse/dep5`

I took option 1.

**Comment by wjt** [comment]
Retest this please

**Pull request merged**